ArduPlane/mode_manual.cpp

```diff
@@ -7,6 +7,25 @@ void ModeManual::update()
     SRV_Channels::set_output_scaled(SRV_Channel::k_elevator, plane.pitch_in_expo(false));
     plane.steering_control.steering = plane.steering_control.rudder = plane.rudder_in_expo(false);
 
+    SRV_Channels::set_output_scaled(SRV_Channel::k_rudder, plane.rudder_in_expo(false));
+    float throttle = plane.get_throttle_input(true);
+    SRV_Channels::set_output_scaled(SRV_Channel::k_throttle, throttle);
```

Comment:
You could move this down to after the quadplane stuff and remove the one on line 25. So QP updates the throttle value before its set.

Reply:
done